src/status/absent.py

```diff
@@ -0,0 +1,55 @@
+from src.decorators import event_listener
+from src.containers import UserDict
+from src.functions import get_players
+from src.messages import messages
+from src import channels
+
+__all__ = ["add_absent"]
+
+ABSENT = UserDict() # type: UserDict[users.User, Tuple[str, bool]]
+
+def add_absent(var, target, key, *, private=True):
```

Comment:
For conformance to all other statuses, the key should be a "reason" (aka prefix) so that multiple messages could be sent out and customized using the specified reason.

Remove the private kwarg and make it always send a pm.